doc/build.md

```diff
@@ -50,11 +50,13 @@ Then, we can build and run tests.
 
 ```bash
 go generate ./...
-SQLFLOW_TEST_DB=mysql go test -v -p 1 ./...
+PYTHONPATH=$GOPATH/src/sqlflow.org/sqlflow/python SQLFLOW_TEST_DB=mysql go test -v -q 1 ./...
```

Comment:
I got your point. Thanks. But I think it should be `PYTHONPATH=/sqlflow/python` in the Docker container. Am I correct?

Reply:
Also, it should be `-p 1` instead of `-q 1`.

Reply:
> Also, it should be `-p 1` instead of `-q 1`.

3x && Updated